### PR TITLE
{bio}[GCC/12.3.0] VEP v113.0, Archive-Zip v1.68, Bio-DB-HTS v3.01, Compress-Raw-Zlib v2.213

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)
resolves https://github.com/vscentrum/vsc-software-stack/issues/450

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

**Comment by laraPPr** [comment]
@PetrKralCZ you are just using the DBD-mysql from this https://github.com/easybuilders/easybuild-easyconfigs/pull/21705 pr right? So that should not cause any conflicts?

**Comment by PetrKralCZ** [comment]
> @PetrKralCZ you are just using the DBD-mysql from this #21705 pr right? So that should not cause any conflicts?

@laraPPr It is the exact easyconfig.

**Comment by verdurin** [comment]
```

#   Failed test 'run - count up/downstream (default)'
#   at ./t/Runner.t line 534.
#          got: '2'
#     expected: '1'

#   Failed test 'run - count up/downstream (10000)'
#   at ./t/Runner.t line 551.
#          got: '3'
#     expected: '2'

#   Failed test 'run - count up/downstream (1000,2000)'
#   at ./t/Runner.t line 570.
#          got: '5'
#     expected: '4'
# Looks like you failed 3 tests of 110.
./t/Runner.t .........................................
```

**Comment by verdurin** [comment]
It's the same error I saw on Simon's slightly older version.

**Comment by boegel** [comment]
@bedroge @branfosj @smoors @akesandgren @bartoldeman Can you take this for a spin on your systems, and see if anyone else has the issue that @verdurin saw?

**Comment by branfosj** [comment]
> @bedroge @branfosj @smoors @akesandgren @bartoldeman Can you take this for a spin on your systems, and see if anyone else has the issue that @verdurin saw?

Test reports on the way, but I think it'll be fine since #21772 built for me.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/711f6186d4f43d53cf0b9430efb89e05 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 7 out of 9 (5 easyconfigs in total)
interactive1 - Linux Rocky Linux 8.10, x86_64, AMD EPYC-Milan Processor (zen3), Python 3.6.8
See https://gist.github.com/bedroge/641230d2bc907f28441076791d78ac6a for a full test report.

**Comment by bedroge** [comment]
> Test report by @bedroge **FAILED** Build succeeded for 7 out of 9 (5 easyconfigs in total) interactive1 - Linux Rocky Linux 8.10, x86_64, AMD EPYC-Milan Processor (zen3), Python 3.6.8 See https://gist.github.com/bedroge/641230d2bc907f28441076791d78ac6a for a full test report.

For me the Bio-DB-HTS tests are failing:
```
Test Summary Report
-------------------
t/01bam.t      (Wstat: 132 (Signal: ILL, dumped core) Tests: 0 Failed: 0)
  Non-zero wait status: 132
  Parse errors: Bad plan.  You planned 452 tests but ran 0.
t/02faidx.t    (Wstat: 132 (Signal: ILL, dumped core) Tests: 4 Failed: 0)
  Non-zero wait status: 132
  Parse errors: Bad plan.  You planned 15 tests but ran 4.
t/03cram.t     (Wstat: 132 (Signal: ILL, dumped core) Tests: 0 Failed: 0)
  Non-zero wait status: 132
  Parse errors: Bad plan.  You planned 139 tests but ran 0.
t/04tabix.t    (Wstat: 132 (Signal: ILL, dumped core) Tests: 1 Failed: 0)
  Non-zero wait status: 132
  Parse errors: Bad plan.  You planned 21 tests but ran 1.
t/05vcf.t      (Wstat: 132 (Signal: ILL, dumped core) Tests: 5 Failed: 0)
  Non-zero wait status: 132
  Parse errors: Bad plan.  You planned 177 tests but ran 5.
t/07cramwrite.t (Wstat: 132 (Signal: ILL, dumped core) Tests: 0 Failed: 0)
  Non-zero wait status: 132
  Parse errors: Bad plan.  You planned 22 tests but ran 0.
t/08bamwrite.t (Wstat: 132 (Signal: ILL, dumped core) Tests: 0 Failed: 0)
  Non-zero wait status: 132
  Parse errors: Bad plan.  You planned 22 tests but ran 0.
t/09sam.t      (Wstat: 132 (Signal: ILL, dumped core) Tests: 0 Failed: 0)
  Non-zero wait status: 132
  Parse errors: No plan found in TAP output
t/10leak.t     (Wstat: 132 (Signal: ILL, dumped core) Tests: 3 Failed: 0)
  Non-zero wait status: 132
  Parse errors: Bad plan.  You planned 6 tests but ran 3.
Files=11, Tests=121,  4 wallclock secs ( 0.05 usr  0.05 sys +  1.29 cusr  0.93 csys =  2.32 CPU)
Result: FAIL
Failed 9/11 test programs. 0/121 subtests failed.
```

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
interactive1 - Linux Rocky Linux 8.10, x86_64, AMD EPYC-Milan Processor (zen3), Python 3.6.8
See https://gist.github.com/bedroge/5702ccb1924e3f44f36990a537377c2b for a full test report.

**Comment by PetrKralCZ** [comment]
@bedroge Can you run `perl ensembl-vep/t/Runner.t` to see more details about what went wrong?
You can also provide the result here directly: https://github.com/Ensembl/ensembl-vep/issues/1802

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node300.hydra.os - Linux Rocky Linux 8.10, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/smoors/414c8cefd9044ac4af9a8d1be77d6d93 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node3141.skitty.os - Linux RHEL 9.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.9.18
See https://gist.github.com/boegel/5f7a83267bc9bb3e31337d2ad2934f13 for a full test report.

File: easybuild/easyconfigs/v/VEP/VEP-113.0-GCC-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+exts_defaultclass = 'PerlModule'
+exts_filter = ("perl -e 'require %(ext_name)s'", "")
+
+exts_list = [
+    ('Bio::EnsEMBL::XS', '2.3.2', {
+        'source_urls': ['https://github.com/Ensembl/ensembl-xs/archive'],
+        'sources': ['%(version)s.tar.gz'],
+        'checksums': ['aafc59568cd1042259196575e99cdfeef9c0fb7966e5f915cfaf38c70885ffa5'],
+    }),
+]
```
Comment (boegel) [inline easybuild/easyconfigs/v/VEP/VEP-113.0-GCC-12.3.0.eb:40]:
```suggestion
]

sanity_check_commands = ['vep --help']

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3102.skitty.os - Linux RHEL 9.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.9.18
See https://gist.github.com/boegel/9f41a557b71fbe31390ac7b7a56a8957 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**